cfme/fixtures/networks.py

```diff
@@ -29,18 +29,20 @@ def test_something(targeted_refresh)
                     targeted_refresh.register_target('ref-aaaabb', 'Router named TEST')
     """
     evm_log = '/var/www/miq/vmdb/log/evm.log'
-    needle_template = r'^.*Collection of targets with id.*:ems_ref=>"{}".*$'
```

Comment:
Is there no match if you leave those? I think we should leave them, unless there is a reason why not.